hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -1304,7 +1304,8 @@ public void testSnapshotBackgroundServices()
         compactionLogsDir, numberOfLogFiles, contentLength,
         currentCompactionLog);
 
-    checkIfCompactionBackupFilesWerePruned(sstBackupDir, numberOfSstFiles);
+    // TODO: create Jira to fix and re-enable this test
```

Comment:
Can you please create a jira and attach it here? Or use HDDS-9206? What ever you feel like?

Reply:
I added to PR description. Will add in code as well.